installing/installing_vmc/installing-restricted-networks-vmc-user-infra.adoc

```diff
@@ -0,0 +1,103 @@
+[id="installing-restricted-networks-vmc-user-infra"]
+= Installing a cluster on VMC in a restricted network with user-provisioned infrastructure
+include::modules/common-attributes.adoc[]
+:context: installing-restricted-networks-vmc-user-infra
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on VMware vSphere infrastructure that you provision in a restricted network by deploying it to link:https://cloud.vmware.com/vmc-aws[VMware Cloud (VMC) on AWS].
+
+Once you configure your VMC environment for {product-title} deployment, you use the {product-title} installation program from the bastion management host, co-located in the VMC environment. The installation program and control plane automates the process of deploying and managing the resources needed for the {product-title} cluster.
+
+include::modules/setting-up-vmc-for-vsphere.adoc[leveloffset=+1]
+include::modules/vmc-sizer-tool.adoc[leveloffset=+2]
+
+== vSphere prerequisites
+
+* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a registry on your mirror host] and obtain the `imageContentSources` data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the mirror host, you can use that computer
+to complete all installation steps.
+====
+* Provision xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] for your cluster. To deploy a private image registry, your storage must provide `ReadWriteMany` access modes.
```

Comment:
See previous comment about RWX

Reply:
@jcpowermac do you propose just deleting the second sentence about RWX access modes? Or is there some more to do for this prereq?

Reply:
Furthermore, if RWX is not available, is it still possible to deploy a private image registry?

Reply:
@jcpowermac can you provide clarity for this?

Reply:
I think without the out of tree csi and vsan services they are stuck with: 
https://docs.openshift.com/container-platform/4.7/registry/configuring_registry_storage/configuring-registry-storage-vsphere.html#installation-registry-storage-block-recreate-rollout_configuring-registry-storage-vsphere


Otherwise another CSI provider on top of what is already installed.